features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/p2.inf

```diff
@@ -12,3 +12,5 @@ org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../featur
 org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.apimgt.core_${feature.version}/libs/Saxon-HE-9.5.1-8.jar/,target:${installFolder}/../../../lib/endorsed/Saxon-HE-9.5.1-8.jar,overwrite:true);\
 org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.apimgt.core_${feature.version}/policy_templates/,target:${installFolder}/../../resources/policy_templates,overwrite:true);\
 org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.apimgt.core_${feature.version}/conf_templates/,target:${installFolder}/../../resources/conf/,overwrite:true);\
+org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../../dbscripts/multidc/);\
```

Comment:
Let's make this `multi-dc`

Reply:
fixed  in 9c350d720f56207e41f4a93a3a6d7df9bfc1dd25